must-gather/Dockerfile

```diff
@@ -13,21 +13,32 @@ RUN curl --location --output velero.tgz https://github.com/openshift/velero/arch
     curl --location --output restic.tgz https://github.com/openshift/restic/archive/refs/heads/${RESTIC_BRANCH}.tar.gz && \
     tar -xzvf restic.tgz && cd restic-${RESTIC_BRANCH} && \
     CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -mod=mod -ldflags '-extldflags "-static"' -o /restic github.com/restic/restic/cmd/restic && \
-    cd .. && rm -rf restic.tgz restic-${RESTIC_BRANCH}
+    cd .. && rm -rf restic.tgz restic-${RESTIC_BRANCH} && \
+    curl --location --output oc.tgz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.16/openshift-client-linux-${TARGETARCH}-rhel9.tar.gz && \
```

Comment:
Here you could use stable (which is currently 4.16), this way we won't need to update once 4.17 is there:

```
curl --location --output oc.tgz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux-${TARGETARCH}-rhel9.tar.gz
```

Reply:
I do not like automatically updates

how does prod handles updates for `oc` @rayfordj ?

Reply:
At the moment, manually, until we have an `oc` available from an el9/ubi9 base image at which time we'll then resume updating via `follow_tag` as the base image is updated.  _I'm guessing this likely won't happen until >=v4.17._

Reply:
While I lean towards `stable` to ensure it isn't later forgotten and doesn't rot as has been observed to occur over the years across our repos.  I've no objection for upstream to use either a) latest `stable` or b) latest versioned (such as, `stable-v4.16`).  However, the former will continue to be updated over releases, while the latter will eventually stop being maintained.  Regardless, it isn't something that will be carried downstream. 
:grimacing:

Reply:
could copy oc from container image

Reply:
changed to `stable`

there is a handy image for that, Tiger? ones I found, where not in `registry.access`, so needed auth